app/assets/stylesheets/components/_index-table.scss

```diff
@@ -3,10 +3,6 @@ index-table is for displaying tabular information in the hub views.
 It expects to be the primary content of the given page (full width).
 */
 
-.scrollable-table-wrapper {
-  overflow-x: scroll;
-}
-
```

Comment:
I did a search for this css class - the clients page is the only place it is used.